tracer/src/Datadog.Trace/Configuration/IntegrationId.cs

```diff
@@ -76,5 +76,6 @@ internal enum IntegrationId
         XpathInjection,
         ReflectionInjection,
         Xss,
+        NHibernate,
```

Comment:
Are we using this value anywhere? If not, we should remove it.

Reply:
```
        IastModule.OnSqlQuery(command, IntegrationId.NHibernate);
```
I use it here when reporting the vuln. Do I keep it? With what can I replace it with if I get rid of it?

Reply:
I missed that one. If you are using it, I would keep it.